show/main.py

```diff
@@ -1621,6 +1621,9 @@ def protocol(verbose):
     from .bgp_frr_v6 import bgp
     ipv6.add_command(bgp)
 
+# Import kube commands
+import kube
+
```

Comment:
Change to `from . import kube`. This should fix the LGTM alerts.

Reply:
Thanks Joe! Trying...

Reply:
It didn't solve the problem. Can you refactor your code to use the `add_command` syntax above and move this import to the top of the file, rather than trying to import the entire file inline in the middle? It's not "Pythonic".

Reply:
That would cause LGTM to raise cyclic warning, as kube imports main too. 

The right way to do is to re-org/re-factor the main.py such that, all helper commands go to a common.py. This way kube would just include common.py (not main.py) and main.py would do config.add_command(kubernetes) and all good.

This is a bigger job. I would rather take that as a separate PR, dedicated to re-factor.

Reply:
I completely agree. I have opened an issue to track this: https://github.com/Azure/sonic-utilities/issues/967. However, you're going to cause 2 LGTM alerts either way, so why not do the refactor now to prevent extra work in the future?

Reply:
I did start and then realized (especially show/main.py), it is much more invasive. Then I decided not to clutter this work item, with changes that are not related to it.

I do agree, that we need to this. Thanks for filing the issue.

Reply:
BTW, you might have noticed this already, that two pys importing each other is already there. For an an example show/main.py and show/bgp_frr_v4.py import each other. This is because, bgp_v4_frr uses some helper functions in main.py. In some cases, to avoid, they duplicate the code.

A refactoring would touch many .py files. So it would be a standalone task by itself, as cleanup.

Reply:
Alerts are fixed. Thanks to all suggestions, which helped.